src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,97 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] arrayList;
```

Comment:
```suggestion
    private T[] elements;
```
Let's not call the inner array of ArrayList as arrayList)